pages.pt_PT/common/touch.md

```diff
@@ -0,0 +1,20 @@
+# touch
+
+> Atualizar as timestamps de um ficheiro para a hora atual.
+> Se o ficheiro não existir, cria um ficheiro vazio, a menos que seja passado o parâmetro -c ou -h.
+
+- Criar um novo ficheiro vazio, ou atualizar as timestamps para a hora atual:
+
+`touch {{ficheiro}}`
+
+- Define as timestamps de um ficheiro para a hora especificada:
```

Comment:
É preciso mudar em todos os exemplos de todas as páginas para usar o infinitivo.

Reply:
Submitted as part of #4235.